charts/lagoon-remote/values.yaml

```diff
@@ -113,6 +113,17 @@ kubernetesBuildDeploy:
     # If not set, a name is generated using the fullname template.
     name:
 
+# sshCore creates a restricted, non-expiring ServiceAccount token for use by
+# lagoon-core.
+sshCore:
+  enabled: true
```

Comment:
> long term

In the short term the core service is the only thing we support in all our tools and is documented, so my rational was to keep the status quo for those that are installing lagoon "today" and we can flip the defaults when ssh-portal is the new standard. I could be convinced otherwise, we'll need to update the docs anyway.

Reply:
sure, i get that. but training people now to acknowledge that this is potentially going away makes more sense.

it you change it from true to false in the future, if people haven't set it to `true` from the start in their own values, then we take the token away from them the next time they helm apply, or have to build some hack logic to not take it off them.

this is the main reason why it should be `enabled: false` and clearly describe why